src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/OcrValidationServerSideException.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
+
+import org.springframework.web.client.HttpServerErrorException;
+
+public class OcrValidationServerSideException extends RuntimeException {
```

Comment:
What do you mean by server side? can you add javadoc?

Reply:
Added JavaDoc.